index.js

```diff
@@ -727,27 +730,40 @@ var tick = function() {
 				if(!("units" in node)) {
 					node.units = {};
 				}
-				if(nodeWinner == node.owner || node.owner == -1 || nodeWinner === -1) {
-					if(node.owner != -1) {
-						nodeWinner = node.owner;
-					}
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["yoMomsHouse" is already defined](https://www.codacy.com/app/vpzomtrrfrt/turbo-spork-server/file/2612211153/issues/source?bid=3248399&fileBranchId=3307983#l736)